src/backend/src/utils/tasks.utils.ts

```diff
@@ -14,3 +15,93 @@ export const convertTaskStatus = (status: Task_Status): TaskStatus =>
     IN_PROGRESS: TaskStatus.IN_PROGRESS,
     DONE: TaskStatus.DONE
   }[status]);
+
+export const hasPermissionToEditTask = async (user: User, taskId: string): Promise<boolean> => {
+  if (user.role === Role.ADMIN || user.role === Role.APP_ADMIN) {
+    return true;
+  }
+
+  const task = await prisma.task.findUnique({
+    where: { taskId },
+    select: {
+      createdByUserId: true,
+      wbsElement: {
+        select: {
```

Comment:
I created this util function to check if a user has permission to edit a task. Should I create a separate test for this function? Currently I onnly have tests for `TasksService.editTaskStatus` which calls `hasPermissionToEditTask`

Reply:
probably a good idea if you don't mind doing it really quick but if it takes too long we can skip it tbh